src/main/java/pl/joegreen/sergeants/simulator/Tile.java

```diff
@@ -0,0 +1,46 @@
+package pl.joegreen.sergeants.simulator;
+
+import java.util.Optional;
+
+interface Tile {
+
+    int TILE_EMPTY = -1;
```

Comment:
I think dependencies from simulator to framework are fine, I would try to avoid the other way dependencies (framework should never depend on simulator) :-) So I would modify and use the existing enum, but it's not a strong opinion, so just do whatever you think is better.